imqsauth.go

```diff
@@ -234,6 +251,221 @@ func createDB(config *authaus.Config) bool {
 	return true
 }
 
+func healthCheck(ic *auth.ImqsCentral, cleanAll bool) bool {
+	db := ic.Central.GetRoleGroupDB()
+	sb := strings.Builder{}
+	w := tabwriter.NewWriter(&sb, 0, 8, 1, '\t', tabwriter.AlignRight)
+	errorSet := map[string]error{}
+	_, err := fmt.Fprintf(w, "\n")
+	if err != nil {
+		// do nothing
+	}
```

Comment:
So, either we return `false`, because we apparently can't log anything to the new writer (which this function heavily depends on), or this should just be
```suggestion
	fmt.Fprintf(w, "\n")
```

Reply:
Yeah this was mostly because intellij moaned...will try again.